api/message_test.go

```diff
@@ -366,18 +366,36 @@ func (s *MessageSuite) Test_CreateMessage_failWhenNoMessage() {
 	assert.False(s.T(), s.notified)
 }
 
-func (s *MessageSuite) Test_CreateMessage_failWhenNoTitle() {
+func (s *MessageSuite) Test_CreateMessage_WhenNoTitle() {
```

Comment:
WhenNoTitle doesn't really make sence
```suggestion
func (s *MessageSuite) Test_CreateMessage_WithoutTitle() {
```